File: docs/SDK-Documentation/getting-started/import-and-initialize/swift/import.swift
import ABSmartly

let sdk: ABSmartlySDK
do {
    let clientConfig = ClientConfig(
        apiKey: ProcessInfo.processInfo.environment["ABSMARTLY_API_KEY"] ?? "",
        application: ProcessInfo.processInfo.environment["ABSMARTLY_APPLICATION"] ?? "",
        endpoint: ProcessInfo.processInfo.environment["ABSMARTLY_ENDPOINT"] ?? "",
        environment: ProcessInfo.processInfo.environment["ABSMARTLY_ENVIRONMENT"] ?? ""))

    let client = try DefaultClient(config: clientConfig)
    let sdkConfig = ABSmartlyConfig(client: client)
    sdk = try ABSmartlySDK(config: sdkConfig)
} catch {
    print(error.localizedDescription)
    return
}

File: docs/SDK-Documentation/getting-started/import-and-initialize/python/import.py
def main():
    client_config = ClientConfig()
    client_config.endpoint = "https://sandbox.test.io/v1"
    client_config.api_key = "test"
    client_config.application = "www"
    client_config.environment = "prod"

    default_client_config = DefaultHTTPClientConfig()
    default_client = DefaultHTTPClient(default_client_config)
    sdk_config = ABSmartlyConfig()
    sdk_config.client = Client(client_config, default_client)
    sdk = ABSmartly(sdk_config)

    context_config = ContextConfig()
    ctx = sdk.create_context(context_config)
